gems/smithy/lib/smithy/views/client/plugin.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Views
+    module Client
+      # @api private
+      class Plugin
+        def initialize(options = {})
+          @class_name = options[:class_name]
+          @require_path = options[:require_path]
+          @default = options.fetch(:default, false)
+          @relative_path = options.fetch(:relative_path, false)
+          @requirable = options.fetch(:requirable, false)
+        end
+
+        attr_reader :class_name, :require_path
+
+        def docstrings
+          docstrings = []
+          options = Object.const_get(@class_name).options
+          options.each do |option|
```

Comment:
In V3 we sometime used "undocumented" options - we may want to support that here as well and do a check to see if the option is documented.

Reply:
Fixed